text_localization_environment/TextLocEnv.py

```diff
@@ -1,65 +1,82 @@
 import gym
-from gym import spaces
 import random
+import numpy as np
+from gym import spaces
 from gym.utils import seeding
 from chainer.backends import cuda
 from PIL import Image, ImageDraw
 from PIL.Image import LANCZOS, MAX_IMAGE_PIXELS
-import numpy as np
-from text_localization_environment.ImageMasker import ImageMasker
+from .ImageMasker import ImageMasker
+from .transformer import create_bbox_transformer
+from .utils import scale_bboxes
 
 
 class TextLocEnv(gym.Env):
     metadata = {'render.modes': ['human', 'rgb_array', 'box']}
 
-    DURATION_PENALTY = 0.03
+    # Length of history in state
     HISTORY_LENGTH = 10
-    # ⍺: factor relative to the current box size that is used for every transformation action
-    ALPHA = 0.2
-    # η: Reward of the trigger action
+
+    # Base Reward for trigger action
     ETA = 7.0
-    # p: Probability for masking a bounding box in a new observation (applied separately to boxes 0..N-1 during premasking)
+    # Base Reward for termination action
+    ETA2 = 10.0
```

Comment:
ETA2 is a bad constant name, either use another character or use ETA_TRIGGER and ETA_TERMINATION.